src/contract/ContractEditor.tsx

```diff
@@ -149,7 +152,11 @@ const renderInput = ({
     )
   }
 
-  if (key === 'procurementUnitIds' && contract.id && contract.startDate && contract.endDate) {
+  if (key === 'procurementUnitIds') {
+    if (isNew) {
+      return <div />
```

Comment:
```suggestion
      return <React.Fragment />
```

Is the way to return an empty thing which is still a React element.